.docs/content/0.armonik/1.glossary.md

```diff
@@ -0,0 +1,37 @@
+# Glossary
+
+**Client:** User-developed software that communicates with the ArmoniK Control Plane to submit a list of tasks to be executed (by one or several workers). It also retrieves results and any error.
+
+**Compute Plane:** Expression designating the set of compute pods, i.e. the pods running a Polling/Scheduling Agent + Worker pair within the Kubernetes cluster.
+
+**Container:** A container is a standard unit of software that packages up code and all its dependencies so the application runs quickly and reliably from one computing environment to another. (see Docker (documentation)[https://www.docker.com/resources/what-container/])
+
+**Control Plane:** Expression designating a set of applications (distributed in different pods) that maintain the Kubernetes cluster as well as communications between certain components (queue, Redis, MongoDB) and the *client*.
+
+**Data dependency:** Input data for a given task that depends on another unique task. Data dependencies formalize dependencies between tasks.
+
+**Data Plane:** Expression designating the set of software components running the various storage and database systems within ArmoniK.
+
+**Kubernetes:** Kubernetes is an open source container orchestration engine for automating deployment, scaling, and management of containerized applications. The open source project is hosted by the Cloud Native Computing Foundation ((CNCF)[https://www.cncf.io/about]) (see Kubernetes (documentation)[https://kubernetes.io/docs/home/#:~:text=Kubernetes%20is%20an%20open%20source,and%20management%20of%20containerized%20applications.]).
+
+**MongoDB:** MongoDB is a document database designed for ease of application development and scaling (see MongoDB (documentation)[https://www.mongodb.com/docs/manual/]).
+
+**Node/Nœud:** Kubernetes runs a workload by placing containers into Pods to run on Nodes. A node may be a virtual or physical machine, depending on the cluster. Each node is managed by the Kubernetes control plane and contains the services necessary to run Pods. (see Kubernetes (documentation)[https://kubernetes.io/docs/concepts/architecture/nodes/]). 
```

Comment:
```suggestion
**Node:** Kubernetes runs a workload by placing containers into Pods to run on Nodes. A node may be a virtual or physical machine, depending on the cluster. Each node is managed by the Kubernetes control plane and contains the services necessary to run Pods. (see Kubernetes [documentation}(https://kubernetes.io/docs/concepts/architecture/nodes/)).
```

Reply:
I don't see why the definition starts with Kubernetes. Start with " an node may be..." and add the first sentence at the end of the description.